### DISCUSSION TITLE
Details About Participating Orgs in GitHub externship

### DISCUSSION THREAD

**Discussion by uzaxirr** [github/india#583] [Students]
Hey There!!!
As I can see that orgs for this cohort are listed https://externship.github.in/organization/list
How can I find more info on their tech stack and the work they do?

**Comment by Siddhant-K-code** [comment]
You Can Manually Go to their GitHub Org. Page and Explore their TechStack. Also, It will be out soon the dashboard itself soon, Once they list their projects for externships.

**Reply by uzaxirr** [reply]
Woww what a coincidence, I just read your [blog](https://dev.to/siddhantkcode/hackerrank-swe-intern-2021-23pi) about hackrrank interview experience yesterday, and here you are replying to my comment 😂

**Reply by Siddhant-K-code** [reply]
Haha! Thanks :)

Hope you liked it :D

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official GitHub Externship Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Reply by mayank-cse** [reply]
Thank you for this discord server 
I have joined it,
I am a second year student can I also take part in this externship program?

**Reply by punitjain03** [reply]
No Mayank, This program is specially curated for 3rd year and 4th year students only
